photon-api/src/main/scala/com/linkedin/photon/ml/optimization/game/RandomEffectOptimizationProblem.scala

```diff
@@ -16,9 +16,11 @@ package com.linkedin.photon.ml.optimization.game
 
 import org.apache.spark.SparkContext
 import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.DataFrame
```

Comment:
This whole class can be deleted.

Reply:
Removed usage of this class in the random effect coordinate.